### ISSUE TITLE
Not able to play certain youtube links 

### ISSUE THREAD

**Issue by tailoric** [lavalink-devs/Lavalink#244]
## Description
whenever I try to play certain songs from youtube lavalink is throwing the following error:
```
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Unable to play this YouTube track.
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.loadTrackFormats(YoutubeAudioTrack.java:140) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.loadBestFormatWithUrl(YoutubeAudioTrack.java:98) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioTrack.process(YoutubeAudioTrack.java:62) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.track.playback.LocalAudioTrackExecutor.execute(LocalAudioTrackExecutor.java:90) ~[lavaplayer-1.3.19.jar!/:na]
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$executeTrack$1(DefaultAudioPlayerManager.java:349) [lavaplayer-1.3.19.jar!/:na]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_222]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_222]
Caused by: java.lang.IllegalStateException: No adaptive formats, no dash, no stream map.
	... 8 common frames omitted
```

Two songs that always result in this error for me (as means of bug reproduction):
https://www.youtube.com/watch?v=bwNV7TAWN3M  
https://www.youtube.com/watch?v=egUkkRpRf2A  

However these videos get played without problems:  
https://www.youtube.com/watch?v=h2P_hqhu5rQ  
https://www.youtube.com/watch?v=v1vXC-vKgKg  
## Version info
**Client used:**
master ver. 1.3.19
Output of `java -jar Lavalink.jar --version`:
```       .   _                  _ _       _    __ _ _                                                      
      /\\ | | __ ___   ____ _| (_)_ __ | | __\ \ \ \                                                     
     ( ( )| |/ _` \ \ / / _` | | | '_ \| |/ / \ \ \ \                                                    
      \\/ | | (_| |\ V / (_| | | | | | |   <   ) ) ) )                                                   
       '  |_|\__,_| \_/ \__,_|_|_|_| |_|_|\_\ / / / /                                                    
    =========================================/_/_/_/                                                     
                                                                                                         
        Version:        3.2.1.1                                                                          
        Build:          828                                                                              
        Build time:     31.07.2019 11:02:24 CEST                                                         
        Branch          master                                                                           
        Commit:         d9c9aa0                                                                          
        Commit time:    31.07.2019 10:59:59 CEST                                                         
        JVM:            1.8.0_232                                                                        
        Lavaplayer      1.3.19           

```
[spring.log](https://github.com/Frederikam/Lavalink/files/3982673/spring.log)

**Comment by freyacodes** [comment]
Please update to the (newly-released) 3.2.2.

If it still doesn't work, you are dealing with a Lavaplayer issue, hence why I am closing this.

https://github.com/sedmelluq/lavaplayer

**Comment by tailoric** [comment]
Update fixed it thanks
